dockerCompose.yml

```
@@ -0,0 +1,46 @@
+version: "3.8"
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)